web/src/main/java/org/akaza/openclinica/service/ImportServiceImpl.java

```diff
@@ -1389,5 +1438,9 @@ private Object validateFormLayout(FormDataBean formDataBean, EventDefinitionCrf
         return formLayout;
     }
 
+    private DiscrepancyNoteService getDiscrepancyNoteService() {
+        discrepancyNoteService = this.discrepancyNoteService != null ? discrepancyNoteService : new DiscrepancyNoteService(this.dataSource);
+        return discrepancyNoteService;
+    }
 
 }
```

Comment:
use @Autowired at the top of the class to instantiate services

Reply:
did code changes based on code review -- now use QueryService